patterns/1-initial/secure-innersource.md

```diff
@@ -0,0 +1,152 @@
+# Title
+
+Secure InnerSource
+
+# Patlet
+
+While InnerSource flourishes in environments with a high degree of shared code, Security/Legal prefers the limitation of source code access to only those that need it.
+By making Security/Legal part of the team, introducing explicit sharing levels, and defining what qualifies as sensitive information, code sharing can be facilitated while minimizing the associated risks.
```

Comment:
@bartgolsteijn I tried myself on a shortened Patlet. What do you think of it?

Reply:
@spier I like the shorter patlet.

Consider adding  [...] introducing explicit sharing levels **and security policies for shared repostories**, and defining what [...]

This covers the the 'SCM setup' and 'Prevent sensitive information in shared repositories' sections. If we leave this out, people might skip the pattern for having too little substance.